README.md

```diff
@@ -1,4 +1,309 @@
+
+4. **Calculating Bad Loan Percentages:**
+   - The percentage of bad loans within each grade was calculated. This is the ratio of bad loans to the total number of loans in each grade, multiplied by 100.
+
+5. **Creating the Bar Plot:**
+   - The bar plot was created using Plotly Express. The x-axis represents the loan grades (A to G), and the y-axis represents the interest rate. The bars are color-coded by the loan condition ('Good Loan' in blue and 'Bad Loan' in red). The `barmode='group'` argument was used to group the bars by loan condition within each grade.
+
+6. **Annotating Bad Loan Percentages:**
+   - Annotations were added to display the percentage of bad loans for each grade. The annotation text is positioned slightly above the midpoint of the bad loan bar for each grade.
+
+### **Results & Insights:**
+- **Interest Rates:**
+  - Interest rates generally increase as the loan grade decreases (from A to G). This trend is expected, as lower-grade loans are riskier, leading to higher interest rates.
+  
+- **Bad Loan Percentages:**
+  - The percentage of bad loans increases with the grade, with Grade A having the lowest percentage (4.7%) and Grade G having the highest percentage (22.2%). This indicates that lower-grade loans not only have higher interest rates but are also more likely to default.
+
+- **Loan Condition:**
+  - The proportion of bad loans is visibly higher in the lower grades, which is highlighted by the increasing size of the red bars as we move from grade A to G.
+
+This visualization effectively communicates the relationship between loan grade, interest rate, and loan condition, showing that both interest rates and the likelihood of a loan being classified as "Bad" increase as the loan grade decreases.
+
+[This and others Visuals from Oleg Movchaniuk](https://github.com/movcha/team_project/blob/main/4_visuals_from_movcha.ipynb)
+
+<br>
+
+---
+
+## Rules of Engagement
+
+1. Transparent and clear communication among team members.
+2. Collaboration and teamwork.
+3. Respect and professionalism in sharing different viewpoints.
+4. Leveraging each team member's expertise for effective project contributions.
+5. Conducting regular team meetings and summarizing key discussion points.
+6. Open idea-sharing.
+7. Providing support for team members facing difficulties.
+8. Commitment to continuous improvement.
+
+---
+
+## Video Links
+
+- [Doga Memis](https://drive.google.com/file/d/1fSrtiI2qsf8S9FBjv8rG0reGXyvNAIt-/view?usp=drive_link)
+- [Jinrong Liu](https://vimeo.com/1000558863/d0e722b4f2?share=copy)
+- [Krishna Kishore](https://vimeo.com/1000959222/4afd8dce93?share=copy)
+- [Kristina Talalaievska](https://drive.google.com/file/d/1hgNK0Y32LRxVpXNiHaegi-lkWQNTQ1SH/view?usp=share_link)
+- [Oleg Movchaniuk](https://drive.google.com/file/d/1QZCuAXnPBWdo4xOdTqFBI4Euz-cY5YA1)
+
+## Sources and references utilized for this project 
+  - [Data folder](./data/)
+  - [Source Code](./src/)
+
+<br>
+
+---
+
+<details>
```

Comment:
I'm a little confused why this is here, team project 1 seems to be the same as team project 2, so why is it separated? I'm not complaining, I like both, but just curious.